README.md

```diff
@@ -6,6 +6,22 @@
 Braid is a simple tool to help track vendor branches in a
 [Git](http://git-scm.com/) repository.
 
+## Fork motivation
```

Comment:
While I can imagine that it would be helpful to readers to have this text in the readme of your fork repository, it doesn't make sense to merge it into upstream Braid.  Please remove this change from the pull request.  If you want to make a readme change like this in the future, please use separate branches in your repository for the readme change and the pull request.

Reply:
Indeed, the readme changes are not relevant, sorry for that.

Reply:
I closed this PR and create another one without readme file changes.